core/cmd/shell.go

```diff
@@ -160,11 +160,17 @@ func (n ChainlinkAppFactory) NewApplication(ctx context.Context, cfg chainlink.G
 
 	loopRegistry := plugins.NewLoopRegistry(appLggr, cfg.Tracing())
 
+	// TOML config + env var protected usage of grpc mercury pool
+	var mercuryTlsCertFile *string
+	if os.Getenv("CHAINLINK_MERCURY_USE_GRPC") == "true" {
+		tlsCertFile := cfg.Mercury().TLS().CertFile()
+		mercuryTlsCertFile = &tlsCertFile
```

Comment:
Do we need to null check here similar to https://github.com/smartcontractkit/chainlink/pull/12560/files#diff-ca252fec8b919ed465bc5f042afca2b4f6be3f20640d327478983967563fb886R357-R359?